x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/xpack/searchablesnapshots/cache/PersistentCache.java

```diff
@@ -555,8 +547,8 @@ private static String getValue(Document document, String fieldName) {
 
     private static CacheKey buildCacheKey(Document document) {
         return new CacheKey(
-            new SnapshotId(getValue(document, SNAPSHOT_NAME_FIELD), getValue(document, SNAPSHOT_ID_FIELD)),
-            new IndexId(getValue(document, INDEX_NAME_FIELD), getValue(document, INDEX_ID_FIELD)),
+            getValue(document, SNAPSHOT_ID_FIELD),
+            getValue(document, INDEX_NAME_FIELD),
```

Comment:
Maybe rename this to SNAPSHOT_INDEX_NAME_FIELD?

Reply:
Good idea